packages/desktop-client/src/components/budget/BudgetTotals.tsx

```diff
@@ -54,11 +56,11 @@ export const BudgetTotals = memo(function BudgetTotals({
           WebkitUserSelect: 'none',
         }}
       >
-        <View style={{ flexGrow: '1' }}>Category</View>
+        <View style={{ flexGrow: '1' }}>{t('Category')}</View>
```

Comment:
Looks good now! Sorry for one last comment, the convention we've been using in the rest of the code base for components is to use `<Trans>`. I.e.:
```suggestion
        <View style={{ flexGrow: '1' }}><Trans>Category</Trans></View>
```
You'll need to import it as well

Reply:
Yeah there are discussion around those to different stage on discord. I did see real statement yet but I'm might be wrong, that why I use the lazy way using one unique code style :p 

However reading the doc it is clearer for me I have to use <Trans> component in some cases: https://actualbudget.org/docs/contributing/i18n

I'll rework the file. Thanks for your heads up.